include/deal.II/physics/elasticity/standard_tensors.h

```diff
@@ -0,0 +1,360 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii__elasticity__standard_tensors_h
+#define dealii__elasticity__standard_tensors_h
+
+
+#include <deal.II/base/symmetric_tensor.h>
+#include <deal.II/base/tensor.h>
+
+DEAL_II_NAMESPACE_OPEN
+
+namespace Physics
+{
+
+  namespace Elasticity
+  {
+
+    /**
+     * A collection of tensor definitions that conform to notation used in
+     * standard scientific literature. These hold specifically for the
+     * codimension 0 case, where the metric tensor is the identity tensor.
+     *
+     * @relates Tensor
+     * @relates SymmetricTensor
+     * @author Jean-Paul Pelteret, Andrew McBride, 2016
+    */
+    template <int dim>
+    class StandardTensors
+    {
+    public:
+
+      /**
+       * @name Metric tensors
+       */
+//@{
+
+      /**
+       * The second-order referential/spatial symmetric identity (metric) tensor
+       * $\mathbf{I}$.
+       *
+       * This is defined such that, for any second-order Tensor or SymmetricTensor,
+       * the following holds:
+       * @f[
+       *  \mathbf{I} \cdot \{ \bullet \} = \{ \bullet \} \cdot \mathbf{I} =
+       * \{ \bullet \}
+       *    \qquad \text{and} \qquad
+       * \mathbf{I} : \{ \bullet \} = \textrm{trace} \{ \bullet \} \, .
+       * @f]
+       * This definition aligns with the second-order SymmetricTensor returned by
+       * unit_symmetric_tensor().
+       */
+      static const SymmetricTensor<2, dim> I;
+
+      /**
+       * The fourth order referential/spatial unit SymmetricTensor $\mathcal{I}$.
```

Comment:
After a conversation with @davydden , I now understand your concern here. I've denoted all fourth order tensors with calliagraphic font (Wriggers), but of course there's no `cal` in variable names. There's also already an `I`, so `II` had to do. I've made a note of how we distinguish vectors/rank-2 tensors/rank-4 tensors notationally the introduction to the physics namespace.  Do you find this satisfactory?